junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/config/JupiterConfiguration.java

```diff
@@ -48,6 +51,8 @@ public interface JupiterConfiguration {
 	String DEFAULT_TEST_CLASS_ORDER_PROPERTY_NAME = ClassOrderer.DEFAULT_ORDER_PROPERTY_NAME;;
 	String DEFAULT_TEST_INSTANTIATION_EXTENSION_CONTEXT_SCOPE_PROPERTY_NAME = ExtensionContextScope.DEFAULT_SCOPE_PROPERTY_NAME;
 
+	Predicate<Class<? extends Extension>> createExtensionFilterByPatterns();
```

Comment:
I think the name should indicate that the filter is only applied to auto-detected extensions.

```suggestion
	Predicate<Class<? extends Extension>> getFilterForAutoDetectedExtensions();
```